dashboard/helm-chart/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: dashboard-lmos-demo
+description: A Helm chart for a dashboard for the LMOS demo, based on Homer
+type: application
+version: 0.1.0
```

Comment:
Should this be `0.1.0-SNAPSHOT`?

Reply:
I guess you are right - looking at https://github.com/eclipse-lmos/arc/issues/114 and assuming that we will do the same for lmos, we should have snapshot versions in the repos. Will change it.